ldk-server-cli/src/main.rs

```diff
@@ -101,6 +102,29 @@ enum Commands {
 		push_to_counterparty_msat: Option<u64>,
 		#[arg(long)]
 		announce_channel: bool,
+		// Channel config options
+		#[arg(long, help = "Forwarding fee proportional (in millionths of a satoshi)")]
+		forwarding_fee_proportional_millionths: Option<u32>,
+		#[arg(long, help = "Forwarding fee base (in milli-satoshi)")]
+		forwarding_fee_base_msat: Option<u32>,
+		#[arg(long, help = "CLTV expiry delta")]
+		cltv_expiry_delta: Option<u32>,
+		#[arg(long, help = "Force close avoidance max fee (in satoshis)")]
+		force_close_avoidance_max_fee_satoshis: Option<u64>,
+		#[arg(long, help = "Accept underpaying HTLCs")]
+		accept_underpaying_htlcs: Option<bool>,
+		#[arg(
+			long,
+			help = "Max dust HTLC exposure as fixed limit (in milli-satoshi)",
+			conflicts_with = "max_dust_htlc_fee_rate_multiplier"
+		)]
+		max_dust_htlc_fixed_limit_msat: Option<u64>,
+		#[arg(
+			long,
+			help = "Max dust HTLC exposure as fee rate multiplier",
+			conflicts_with = "max_dust_htlc_fixed_limit_msat"
+		)]
+		max_dust_htlc_fee_rate_multiplier: Option<u64>,
```

Comment:
Some of these options could really use some more documentation, but at the same time, I wouldn't expect users to set these in most cases. Maybe we just keep the forwarding specific ones for now and revisit the others if there's interest from users?